index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+   
+  </head>
+
+  <body>
+   <header id="title">Wangari Maathai</header>
```

Comment:
- [ ]  I think it would be better to use inline comments under the portion of the code you want the student to make changes or modify. This would assist students to easily locate areas their attention is being called to. It is one of the most powerful features GitHub code review has. This example demonstrate how to use it in your future review.